examples/hybrid/sphere/deformation_flow.jl

```diff
@@ -280,52 +289,73 @@ tracer_ranges(sol) =
         return maximum(q_n) - minimum(q_n)
     end
 
-ref_sol = run_deformation_flow(false, nothing)
+third_sol = run_deformation_flow(false, upwind3)
```

Comment:
Suggest changing varname to `upwind_first_sol` or similar so its consistent with the `centered_sol`, `fct_sol` etc (seems clearer to me)